src/Grid.js

```diff
@@ -165,7 +169,7 @@ Grid.prototype.redraw = function(newOptions, callback) {
   var items;
 
   window.requestAnimationFrame(function() {
-    if (this.columns !== newOptions.columns) {
+    if (this.columns !== newOptions.columns || this.columnClasses !== newOptions.columnClasses) {
```

Comment:
I agree that redraws **may** be quite expensive. But they occurs rarely (not on every `mousemove` event for example). So my first intension was to not complicate logic with edge cases and always do redraws. If you wish, I can optimise this by changing css classes only instead of full redraw.